### ISSUE TITLE
Can't bind to 'eventSource' since it isn't a known property of 'calendar'.

### ISSUE THREAD

**Issue by theunreal** [twinssbc/Ionic2-Calendar#110]
Latest version, library not working

`Error: Template parse errors: Can't bind to 'eventSource' since it isn't a known property of 'calendar'.`

**Comment by mharrisn** [comment]
@theunreal How did you resolve this issue? I'm experiencing the same error. I have the CalendarModule added.

**Comment by danielfnz** [comment]
@twinssbc I have the same error when build --prod;

Using "ionic server" its ok;

**Comment by twinssbc** [comment]
@danielfnz Did you add NgCalendarModule in your app.module files?

```
imports: [
        NgCalendarModule,
        IonicModule.forRoot(MyApp)
    ],
```

**Comment by danielfnz** [comment]
Yes, i did.

Em 29 de jun de 2017 4:58 AM, twinssbc <notifications@github.com> escreveu:

@danielfnz<https://github.com/danielfnz> Did you add NgCalendarModule in your app.module files?

imports: [
        NgCalendarModule,
        IonicModule.forRoot(MyApp)
    ],

**Comment by twinssbc** [comment]
@danielfnz Do you have main.prod.ts file? Do you add the below lines to it according to README?

```
import { CalendarComponent } from 'ionic2-calendar/calendar';
import { MonthViewComponent } from 'ionic2-calendar/monthview';
import { WeekViewComponent } from 'ionic2-calendar/weekview';
import { DayViewComponent } from 'ionic2-calendar/dayview';
```

**Comment by gerardob** [comment]
hi @twinssbc I have the same issue when try to use --prod; I have the:
 
import { CalendarComponent } from 'ionic2-calendar/calendar';
import { MonthViewComponent } from 'ionic2-calendar/monthview';
import { WeekViewComponent } from 'ionic2-calendar/weekview';
import { DayViewComponent } from 'ionic2-calendar/dayview';

in the main.ts it's look like:

import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';

import { CalendarComponent } from 'ionic2-calendar/calendar';
import { MonthViewComponent } from 'ionic2-calendar/monthview';
import { WeekViewComponent } from 'ionic2-calendar/weekview';
import { DayViewComponent } from 'ionic2-calendar/dayview';

import { AppModule } from './app.module';

platformBrowserDynamic().bootstrapModule(AppModule);

also in the app.module.ts I have the import:

import { NgCalendarModule } from 'ionic2-calendar';

the complete file is:

import { BrowserModule } from '@angular/platform-browser';
import { FormsModule }   from '@angular/forms';
import { HttpModule } from '@angular/http';
import { ErrorHandler, NgModule } from '@angular/core';
import { IonicApp, IonicErrorHandler, IonicModule } from 'ionic-angular';
import { SplashScreen } from '@ionic-native/splash-screen';
import { StatusBar } from '@ionic-native/status-bar';
import { EmailComposer } from '@ionic-native/email-composer';
import {LOCALE_ID} from '@angular/core';


import { MyApp } from './app.component';
import { LoginPageModule } from '../pages/login/login.module';
import { CalendarPageModule } from '../pages/calendar/calendar.module';
import { DetailsPageModule } from '../pages/details/details.module';
import { SummaryPageModule } from '../pages/summary/summary.module';
import { OptionsPageModule } from '../pages/options/options.module';
import { NgCalendarModule } from 'ionic2-calendar';
import { ApiProvider } from '../providers/api/api';


@NgModule({
  declarations: [
    MyApp
  ],
  imports: [
    NgCalendarModule,
    BrowserModule,
    FormsModule,
    HttpModule,
    CalendarPageModule,
    SummaryPageModule,
    DetailsPageModule,
    LoginPageModule,
    OptionsPageModule,
    IonicModule.forRoot(MyApp)
  ],
  bootstrap: [IonicApp],
  entryComponents: [
    MyApp
  ],
  providers: [
    StatusBar,
    SplashScreen,
    {provide: LOCALE_ID, useValue: 'es-AR'},
    {provide: ErrorHandler, useClass: IonicErrorHandler},
    ApiProvider,
    EmailComposer
  ]
})
export class AppModule {

}


I don't if i need to make any change in the page.module.ts?

**Comment by twinssbc** [comment]
@gerardob Since you are using a page module, you need to add NgCalendarModule in the imports field of your page.module not app.module.

**Comment by gerardob** [comment]
yes thank you!! in the weekend I was testing and I do that and working perfectly.. 

another question not related. where are the default template of the month view event? I need use the same template only I need make a little change but I don't see that in the repo files??

**Comment by twinssbc** [comment]
@gerardob The default template is in https://github.com/twinssbc/Ionic2-Calendar/blob/master/src/calendar.ts
